akka-actor/src/main/scala/akka/io/InetAddressDnsResolver.scala

```diff
@@ -49,3 +57,45 @@ class InetAddressDnsResolver(cache: SimpleDnsCache, config: Config) extends Acto
       sender() ! answer
   }
 }
+
+/**
+ * Mirrors obtaining the settings from Security properties as implemented
+ * in [[sun.net.InetAddressCachePolicy]] features (mostly constants).
+ *
+ * Shadow impl needed in order to not depend on `sun.net` packages, retaining
+ * compatibility with those settings.
+ */
+@InternalApi private object InetAddressCachePolicyShadow {
```

Comment:
and this was intended to be included in this PR?

Reply:
Oh, no; it wasn't; sorry 😞